### ISSUE TITLE
Move potted plant err

### ISSUE THREAD

**Issue by Saturn-99** [UMDLARS/one_night_in_sf#220]
![image](https://github.com/UMDLARS/one_night_in_sf/assets/116676184/74dd1556-8ba5-42fb-92d3-6b7569dc2596)

**Comment by spacehobo** [comment]
What were you hoping for this to do?  The `Transfer` verb in Inform requires a destination to be valid as parsed.

**Comment by pahp** [comment]
Hey @spacehobo this seems somewhat similar to [this issue ](https://github.com/UMDLARS/one_night_in_sf/issues/272) -- I would think that saying `move plant` would result in `Move plant where?` or `You'll have to be more specific.`

**Comment by roell030** [comment]
Moving any object results in the same programming errors:

![Image](https://github.com/user-attachments/assets/3cfec257-1c5c-456c-be2a-548550132831)
![Image](https://github.com/user-attachments/assets/dcf30ac9-f603-464b-9f9c-a1f014906fef)
![Image](https://github.com/user-attachments/assets/a06b99e2-1d5b-4d6d-af51-08d539e002f1)
![Image](https://github.com/user-attachments/assets/1eb8a625-9a01-4593-9a73-b529862a698e)

Also The first `move(object)` results in `@get_prop_addr` and `@get_prop` warnings

**Comment by pahp** [comment]
Yep, closing. Sorry about that... I had no idea that anything could persist after a forced reload.
